aiopyarr/models/request_common.py

```diff
@@ -162,7 +162,7 @@ def __post_init__(self):
         """Post init."""
         if (
             self.sizeleft > 0
-            and self.timeleft == "00:00:00"
+            and (hasattr(self, "timeleft") and self.timeleft == "00:00:00")
             or not hasattr(self, "trackedDownloadState")
```

Comment:
```
            not hasattr(self, "timeleft")
            or self.sizeleft > 0
            and self.timeleft == "00:00:00"
            or not hasattr(self, "trackedDownloadState")
```

Reply:
Can you also add the below to the tests on https://github.com/tkdrob/aiopyarr/blob/07d120c39bed7a1fb7d910b5b4752f962f2a1d17/tests/test_sonarr.py#L639
```
    assert not hasattr(data.records[1], "timeleft")
    assert data.records[1].trackedDownloadState == "stopped"
```